lesson_03/homework_03.py

```diff
@@ -0,0 +1,137 @@
+"""This is a file for Homework 3."""
```

Comment:
Ok Linter shows this:
```
./bin/lint ./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:14:80: E501 line too long (80 > 79 characters)
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:45:57: C813 missing trailing comma in Python 3
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:78:33: C813 missing trailing comma in Python 3
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:99:42: W504 line break after binary operator
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:100:48: W504 line break after binary operator
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:101:34: W504 line break after binary operator
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:102:32: W504 line break after binary operator
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:105:80: E501 line too long (84 > 79 characters)
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:135:80: E501 line too long (82 > 79 characters)
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:136:80: E501 line too long (96 > 79 characters)
./T_HOMETASK/Lesson_03/03_Anna_Chuprina.py:136:97: C813 missing trailing comma in Python 3
```
Please fix all "line too long" errors 
See   
I gave info how to fix such errors here 
https://github.com/antohaUa/AQA_241007_Hillel/blob/main/Lesson_03_Operators_Int_Float/02_long_lines.py and on video Lesson_03 ( start )